vue/sbc-common-components/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM node:10.24.1 as build-stage
+WORKDIR /app
+COPY ./package*.json ./
+RUN npm install
+COPY . .
+RUN npm run build
+
+FROM nginx:1.18.0 as production-stage
+
+ARG VCS_REF="missing"
+ARG BUILD_DATE="missing"
+
+ENV VCS_REF=${VCS_REF}
+ENV BUILD_DATE=${BUILD_DATE}
+
+LABEL org.label-schema.vcs-ref=${VCS_REF} \
+    org.label-schema.build-date=${BUILD_DATE}
+
+COPY nginx.conf /etc/nginx/nginx.conf
+RUN mkdir /app
+COPY --from=build-stage /app/dist /app
+EXPOSE 8080:8080
+CMD ["nginx", "-g", "daemon off;"]
```

Comment:
Not sure if this is relevant, this is a library that's included in other projects. If there is a better way of doing this let me know.

Reply:
I don't think you need a Dockerfile. This package will be published to the NPM repository. and won't run by itself.